scripts/build-tools.js

```diff
@@ -1,18 +1,20 @@
 const { getData } = require('./tools/extract-tools-github');
 const { convertTools } = require('./tools/tools-object');
 const { combineTools } = require('./tools/combine-tools');
+const manualTools = require('../config/tools-manual.json')
```

Comment:
I moved it here from `combine-tools` as it felt confusing for me that we read one of input data inside the helper function

Reply:
rest of changes is just related to it